### DISCUSSION TITLE
Params passed after email verification?

### DISCUSSION THREAD

**Discussion by prescience-data** [supabase/supabase#3042] [Questions] [answered]
Looking at [the docs here](https://supabase.io/docs/reference/javascript/auth-signup#sign-up) it doesn't seem to list the params that Supabase sends to the client after email verification? (are there even any?)

For password resets, I listen for `type=recovery` and `access_token` (which are documented), but not sure if there is something similar for email verification that I can watch for... 🤔

**Comment by kangmingtay** [comment, marked as answer]
Hey @prescience-data, for confirmation emails / email verification, we send a link which contains the following query params:
`type=signup`, `redirect_to=your_redirect_url` and `token=confirmation_token`
